File: C/countNumber.c
#include <stdio.h>
void countNumber(int n){
	int cnt = 0;
	while (n != 0) {
		n /= 10;
		cnt++;
	}
	printf("%d",cnt);
}
int main(){
	int n;
	scanf("%d", &n);
	countNumber(n);
	return 0;
	
}

File: Java/countnumber.java

import java.util.Scanner;

public class countnumber {
	public static void cntNumber(int n) {
		int cnt = 0;
		while (n!=0) {
			cnt++;
			n /= 10;
		}
		System.out.print(cnt);
	}
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		cntNumber(n);
				

	}

}
